templates/noKibana/deploys.go

```diff
@@ -0,0 +1,12 @@
+package noKibana
+
+// Deploys will hold the configuration for the deploys index
+var Deploys = Object{
```

Comment:
I don't like the `Deploys` naming.. please use `ScDeploys` sau `ScOwners` or something like this

Reply:
renamed.